doc/Language/objects.pod6

```diff
@@ -447,10 +444,10 @@ After the C<BUILD> methods have been called, methods named C<TWEAK> are
 called, if they exist, again with all the named arguments that were passed
 to C<new>.
 
-Due to the nature of the default behavior of C<BUILDALL> and custom C<BUILD>
+Due to the default behavior of C<BUILDALL> and C<BUILD>
```

Comment:
This is a run on sentence and should also not start with the word “Due”. You could start it with “The default behavior” which would be much better. Extra points if you break it into more than one sentence.